export const captureException = (error: any) => {};
